sites/feedback/ajax/page-feedback-contact-en.html

```diff
@@ -0,0 +1,69 @@
+<div id="gc-pft" class="row wb-disable-allow" data-wb-jsonmanager='{
+	"name": "feedback",
```

Comment:
considering this dataset name are going to be on several pages, use a more specific name like "gc-pft".